sdcm/cluster.py

```diff
@@ -3203,16 +3204,44 @@ def download_monitor_data(self):
         for node in self.nodes:
             try:
                 # self.stop_scylla_monitoring(node)
-                monitoring_data_dir_path = self.download_monitoring_data_dir(node)
-                # self.start_scylla_monitoring(node)
-                shutil.make_archive(node.logdir, 'zip', monitoring_data_dir_path)
-                zipped_data_path = '%s.zip' % node.logdir
-                return S3Storage.upload_file(zipped_data_path)
+                snapshot_archive = self.get_snapshot(node)
+                self.log.debug('Snapshot local path: {}'.format(snapshot_archive))
+
+                return S3Storage.upload_file(snapshot_archive)
             except Exception, details:
                 self.log.error('Error downloading prometheus data dir: %s',
                                str(details))
         return ""
 
+    def get_snapshot(self, node):
```

Comment:
Currently we use phantomjs to get snapshot, can we use the new graceful method to replace phantomjs?

Reply:
@amoskong we can. If you have any other method that you know please add a task in NEW column in Trello and we will consider replacing phantomJS

Reply:
BTW this method is not related to screenshot that we take. There is a difference between the words `snapshot` and `screenshot`. Snapshots usually used in DB, filesystem context, while screenshot is used to capture screen.
To make things clear, Alex, please rename `get_snapshot` to `get_prometheus_snapshot`

Reply:
Renamed